src/modules/NtpClient.rb

```diff
@@ -1263,6 +1213,44 @@ def AutoPackages
       { "install" => @required_packages, "remove" => [] }
     end
 
+  private
+
+    def read_known_servers
+      servers_file = Directory.find_data_file("ntp_servers.yml")
+
+      return {} if !servers_file
+
+      # FIXME: This is wrong approach, because datadir should be array of all Y2DIR/DATA
+      # so we need to skip it in tests, even if we set Y2DIR properly
+      servers = YAML.load_file(servers_file)
+      if servers.nil?
+        log.error("Failed to read the list of NTP servers")
+        return {}
+      end
+
+      log.info "#{servers.size} known NTP servers read"
```

Comment:
I prefer to have in log real vaues instead of number, it is not much helpful for debugging